ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/disconnecting-network/deprovision/cucushift-installer-rehearse-vsphere-upi-disconnecting-network-deprovision-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-upi-disconnecting-network-deprovision
+  steps:
+  - chain: gather-core-dump
+  - chain: gather
+  - ref: upi-deprovision-vsphere
```

Comment:
I am tidy up deprovision chains in https://github.com/openshift/release/pull/33289, do you mind to replace the above steps with `chain: cucushift-installer-rehearse-vsphere-upi-deprovision`